ncbi/src/main/java/org/reactome/release/EuropePMC.java

```diff
@@ -0,0 +1,375 @@
+		 * @return Reactome Pathway Stable Identifier
+		 */
+		public String getPathwayStableId() {
+			return pathwayStableId;
+		}
+
+		/**
+		 * Sets Reactome Pathway Stable identifier as long as it begins with "R-HSA" indicating it is a human pathway
+		 * @param pathwayStableId Reactome Pathway Stable Identifier
+		 * @throws IllegalArgumentException Throws IllegalArgumentException if the pathway stable id doesn't begin
+		 * with "R-HSA"
+		 */
+		private void setPathwayStableId(String pathwayStableId) {
+			if (!pathwayStableId.startsWith("R-HSA")) {
+				throw new IllegalArgumentException(
+					"Pathway stable id " + pathwayStableId + " does not start with R-HSA " +
+					"(i.e. it is not a human stable id)"
+				);
+			}
+
+			this.pathwayStableId = pathwayStableId;
+		}
+
+		/**
+		 * Retrieves the PubMed identifier for the Link annotation
+		 * @return PubMed identifier as String
+		 */
+		public String getPubMedIdentifier() {
+			return pubMedIdentifier;
+		}
+
+		/**
+		 * Sets PubMed literature identifier as long as it is numeric
+		 * @param pubMedIdentifier PubMed literature identifier
+		 * @throws IllegalArgumentException Throws IllegalArgumentException if the PubMed literature reference isn't
+		 * numeric
+		 */
+		private void setPubMedIdentifier(String pubMedIdentifier) {
+			if (!pubMedIdentifier.matches("\\d+")) {
+				throw new IllegalArgumentException(
+					"PubMed identifier " + pubMedIdentifier + " is not numeric"
+				);
+			}
+
+			this.pubMedIdentifier = pubMedIdentifier;
+		}
+
+		/**
+		 * URL representing the Link XML annotation's Reactome Pathway
+		 * @return Pathway URL as String
+		 */
+		private String getPathwayURL() {
+			return ReactomeConstants.PATHWAY_BROWSER_URL + getPathwayStableId();
+		}
+
+		/**
+		 * Link XML based on pre-set Reactome's Europe PMC provider id and the annotation's pathway name, URL, and
+		 * PubMed literature identifier
+		 * @return Link XML tag as String
+		 */
+		public String getLinkXML() {
+			return String.join(System.lineSeparator(),
+				indentString("<link providerId=\"" + reactomeProviderID + "\">", 1),
+				indentString("<resource>",2),
+				indentString("<title>" + getPathwayDisplayName() + "</title>", 3),
+				indentString("<url>" + getPathwayURL() + "</url>", 3),
+				indentString("</resource>",2),
+				indentString("<record>", 2),
+				indentString("<source>MED</source>", 3),
+				indentString("<id>" + getPubMedIdentifier() + "</id>", 3),
+				indentString("</record>",2),
+				indentString("</link>",1)
+			);
+		}
+
+		/**
+		 * Checks equality based on object type and value of pathway display name, stable id, and PubMed identifier
+		 * @param o Object to check for equality with the calling Europe PMC Link object.
+		 * @return <code>true</code> if the same object or a Europe PMC Link object with the same pathway display name,
+		 * stable, id and PubMed identifier.  Returns <code>false</code> otherwise.
+		 */
+		@Override
+		public boolean equals(Object o) {
```

Comment:
More descriptively named variable

Reply:
I think this been an old Java convention when overriding "equals". But I agree, it's not the prettiest convention. In newer documentation, I see "obj" used instead of "o".

Reply:
I've changed `o` to `obj` so it's a little more verbose.